jee-learning-platform/backend/app/main.py

```
@@ -0,0 +1,168 @@
+from fastapi import FastAPI, HTTPException, Request
```

Ruff (0.11.9)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)